### PR TITLE
People/Add api methods for user removing

### PR DESCRIPTION
**Pull request by olga-nesterova**
Add methods for `People/Remove user data` section 
https://api.onlyoffice.com/portals/method/people/post/api/2.0/people/remove/start
https://api.onlyoffice.com/portals/method/people/get/api/2.0/people/remove/progress
https://api.onlyoffice.com/portals/method/people/put/api/2.0/people/remove/terminate

### CODE REVIEW

File: lib/teamlab/modules/people/people_remove.rb [hunk 1]
```diff
@@ -0,0 +1,27 @@
+# frozen_string_literal: true
+
+module Teamlab
+  # Module for people removing process
+  module PeopleRemove
+    # Start a remove process
+    # @param userId [String] guid of user to remove
+    # @param [Hash] remove progress data
+    def start_remove(user_id)
```
Comment (ShockwaveNN) [inline lib/teamlab/modules/people/people_remove.rb:9]: I think this method can remove some useful info, or nothing and in that case `@return [void]`

File: lib/teamlab/modules/people/people_remove.rb [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+      @request.post(%w[remove start], userId: user_id)
+    end
+
+    # Returns the progress of the started remove process
+    # @param userId [String] guid of user to remove
+    # @param from_user_id [String] remove progress data
+    def get_remove_progress(user_id)
```
Comment (ShockwaveNN) [inline lib/teamlab/modules/people/people_remove.rb:16]: This method defenetly `@return` some data

Comment (ShockwaveNN) [inline lib/teamlab/modules/people/people_remove.rb:16]: In general - better not to name methods starting with `get`

Maybe `def remove_progress(user_id)`

**Review by ShockwaveNN — CHANGES_REQUESTED** [review]

File: lib/teamlab/modules/people/people_remove.rb (hunk 2 above)
Comment (olga-nesterova) [reply lib/teamlab/modules/people/people_remove.rb]: Screen Shot 2021-08-16 at 17 21 30
Changed it but that's how this method is named in the documentation

Comment (ShockwaveNN) [reply lib/teamlab/modules/people/people_remove.rb]: I think we do not need to duplicate documentation name by 100%

Ading `get` is unusual for `ruby` language, not for any language
But we're writing ruby library so I think we should first of all try to be in order with ruby language guidelines, not with strange names in our documentation, like `photoes`))

**Review by ShockwaveNN — APPROVED** [review]

**Pull request merged**